charts/promtail/values.yaml

```diff
@@ -214,6 +214,14 @@ podSecurityPolicy:
   requiredDropCapabilities:
     - ALL
 
+# -- You can put here any keys that will be directly added to the config file's 'client' block.
+# @default -- empty
+extra_client_configs: {}
+
+# -- You can put here any additional scrape configs you want to add to the config file.
+# @default -- empty
+extra_scrape_configs: {}
```

Comment:
If we add something like this, then it must go under `config.snippets`. Also: camel case please.

Reply:
RIght, but this requires overriding the whole `config.file`. I really liked the previous approach, where you could override only some sections and keep your actual config shorter. As for camel case - sure, I'm not sure how I missed that :P

Reply:
OK, I guess it makes sense.